### PR TITLE
[13.0][ADD] new module purchase_stock_analytic

### PR DESCRIPTION
**Pull request by baimont**
New module that copies the analytic account and the analytic tags of the purchase order item to the stock move

Depends on https://github.com/OCA/account-analytic/pull/332

### CODE REVIEW

**Comment by sanrav** [comment]
Functional testing OK

**Review by apineux — APPROVED** [review]

**Review by sanrav — APPROVED** [review]
Functional testing OK

File: purchase_stock_analytic/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    "name": "Purchase Stock Analytic",
+    "summary": """
+        Copies the analytic account of the purchase order item to the stock move""",
+    "version": "13.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "ACSONE SA/NV,Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/account-analytic",
+    "depends": ["purchase", "stock_analytic"],
+    "data": [],
+    "demo": [],
```
Comment (MeetKD) [inline purchase_stock_analytic/__manifest__.py]: @baimont Can you please remove empty keys ?

File: purchase_stock_analytic/models/purchase_order_line.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+class PurchaseOrderLine(models.Model):
+
+    _inherit = "purchase.order.line"
+
+    def _prepare_stock_moves(self, picking):
+        res = super(PurchaseOrderLine, self)._prepare_stock_moves(picking)
+        for line in self:
+            aa = line.account_analytic_id
+            at = line.analytic_tag_ids
```
Comment (MeetKD) [inline purchase_stock_analytic/models/purchase_order_line.py]: Can we please improve the variable names lie aa to analytic_account and at to analytic_tags ?

**Review by MeetKD — CHANGES_REQUESTED** [review]
Minor Changes !

**Review by apineux — APPROVED** [review]

File: purchase_stock_analytic/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+{
+    "name": "Purchase Stock Analytic",
+    "summary": """
+        Copies the analytic account of the purchase order item to the stock move""",
+    "version": "13.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "ACSONE SA/NV,Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/account-analytic",
+    "depends": ["purchase", "stock_analytic"],
```
Comment (rousseldenis) [inline purchase_stock_analytic/__manifest__.py]: @baimont I would say that the exact dependency is 'purchase_stock' as the method you override is defined there.

Comment (rousseldenis) [reply purchase_stock_analytic/__manifest__.py]: @baimont You can take the last two commits of https://github.com/OCA/account-analytic/pull/410

**Review by rousseldenis — APPROVED** [review]

**Comment by rousseldenis** [comment]
@MeetKD

**Review by MeetKD — APPROVED** [review]

**Review by acsonefho — APPROVED** [review]

**Comment by rousseldenis** [comment]
@OCA/accounting-maintainers

**Pull request merged**